relay-server/src/services/buffer/mod.rs

```diff
@@ -77,23 +79,19 @@ impl FromMessage<Self> for EnvelopeBuffer {
 // NOTE: This pattern of combining an Addr with some observable state could be generalized into
```

Comment:
This PR effectively implements this note, right?

Reply:
Good catch, will remove this.